collector/collector.go

```diff
@@ -113,6 +120,17 @@ func (c *EventsCollector) collectEvents(url string) {
 loop:
 	for {
 		select {
+		case data := <-proposals:
```

Comment:
what this collector does? maybe this is some old artifact and we can ditch it instead of maintaining?

Reply:
the only thing is the memory collector used in cmd/multi_node_sim.go. i am leaving it for now. not sure what it's for tho.